rules/community/cloudwatch_events/cloudtrail_critical_api_calls.json

```diff
@@ -846,6 +846,24 @@
     "source": "prefix.cluster.sample.bucket",
     "trigger_rules": [
       "cloudtrail_critical_api_calls"
-    ]
+    ],
+    "publisher_tests": {
+      "slack:sample-channel": [
+        [
+          "keys(@)",
```

Comment:
can we document this a little more so people understand what this refers to

Reply:
also - what are your thoughts on making this list more explicit.. something like:

```json
{
  "publisher_tests": {
    "slack:sample-channel": [
      {
        "jmespath_descriptor": "keys(@)",
        "conditional": "is",
        "value": [
          "@slack.text",
          "@slack.attachments",
          "@slack._previous_publication"
        ]
      },
      {
        "jmespath_descriptor": "\"@slack.text\"",
        "conditional": "is",
        "value": "Rule triggered"
      }
    ]
  }
}
```

Reply:
More explicit, more wordy to write, harder to read, since the "subject-verb-predicate" doesn't make sense anymore.

Reply:
but but but.. usability

Reply:
Compromise: will support **_both_** syntaxes